devices_and_sensors.cpp

```diff
@@ -0,0 +1,26 @@
+#include "devices_and_sensors.h"
+
+void DevicesAndSensors::modifyPrefferedValue(std::string sensor, int value) {
+  numericalDevices[sensor] = value;
+}
+
+void DevicesAndSensors::turnDeviceOnOrOff(std::string device, bool state) {
+  onOffDevices[device] = state;
```

Comment:
If you would have a separate "Device" class, with it's own state member, this would look like:
Device setState(bool state) {
  this->state = state;
}